wdl/HardCloneWorkspace/HardCloneWorkspace.wdl

```diff
@@ -0,0 +1,56 @@
+version 1.0
+
+workflow hard_clone_terra_workspace {
```

Comment:
To keep this consistent with the other WDLs, let's make this `HardCloneTerraWorkspace`